### PR TITLE
Unit tests for line widget

### PR DESCRIPTION
**Pull request by C47D**
### Description of the feature or fix

Unit tests for line widget

### Checkpoints
- [x] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [x] Run `code-format.py` from the `scripts` folder. [astyle](http://astyle.sourceforge.net/install.html) needs to be installed.
- [ ] Update the documentation

### CODE REVIEW

**Comment by C47D** [comment]
I don't see any other behavior to test, maybe adding a rendering test would be useful?

File: tests/src/test_cases/test_line.c [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+void setUp(void)
+{
+    active_screen = lv_scr_act();
+    line = lv_line_create(active_screen);
+}
+
+void tearDown(void)
+{
+    /* Function run after every test */
```
Comment (kisvegabor) [inline tests/src/test_cases/test_line.c]: Adding `lv_obj_clean(lv_scr_act())`?

**Review by kisvegabor — COMMENTED** [review]
I can imagine two more behavior test: 
1. by default the size should be set according to [this](https://github.com/lvgl/lvgl/blob/master/src/widgets/lv_line.c#L152-L153)
2. the extra draw size should be set according to [this](https://github.com/lvgl/lvgl/blob/master/src/widgets/lv_line.c#L137).

**Comment by C47D** [comment]
For extra draw size I should use `lv_obj_refresh_ext_draw_size` on the line object, right? Or how can we test the calculation?

**Comment by kisvegabor** [comment]
`_lv_obj_get_ext_draw_size(obj);` should tell the current ext draw size.

**Comment by C47D** [comment]
@kisvegabor on the bd4a8fc (test for extra draw size) there's something missing, as far as I understand, when the line style line width is greater than the received value from the event, the extra draw size should be set to the style line width. But I'm getting the initial extra draw size (5) and were expecting (10).

File: tests/src/test_cases/test_line.c [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    lv_obj_add_event_cb(line, line_event_cb, LV_EVENT_ALL, NULL);
+    /* Trigger the extra draw size event */
+    lv_obj_refresh_ext_draw_size(line);
+
+    TEST_ASSERT_EQUAL(1U, _lv_obj_get_ext_draw_size(line));
+
+    /* Update line width style */
+    static lv_style_t line_style;
+    lv_style_init(&line_style);
+    lv_obj_set_style_line_width(line, LV_PART_MAIN, final_extra_draw_size);
```
Comment (kisvegabor) [inline tests/src/test_cases/test_line.c]:
```suggestion
    lv_obj_set_style_line_width(line, final_extra_draw_size, LV_PART_MAIN);
```

**Review by kisvegabor — COMMENTED** [review]
I found the issue, it was only tricky typo.

`line_style` related parts are not required, right?

File: tests/src/test_cases/test_line.c (hunk 2 above)
Comment (C47D) [reply tests/src/test_cases/test_line.c]: Good catch, fixed in 8c14fbe

**Comment by C47D** [comment]
@kisvegabor Thanks for the help, now all tests are passing. Ready to be merged.

**Comment by kisvegabor** [comment]
Aaand merged, thank you again! :slightly_smiling_face:

**Pull request merged**